tests/phpunit/src/TestBase.php

```diff
@@ -21,7 +21,8 @@
   public function __construct($name = NULL, array $data = array(), $data_name = '') {
 
     parent::__construct($name, $data, $data_name);
-    $this->projectDirectory = dirname(dirname(dirname(__DIR__)));
+    $directory = realpath(dirname(__FILE__) . '/../../../');
```

Comment:
This is likely failing because of how the tests are called. They are intended to be called from the child `blt-project`, but they are being called from the parent `blt` here:
https://github.com/acquia/blt/blob/8.x/.travis.yml#L90

We'll want to tag these tests with a group like `@blt-project`. Then, we can run test in two sets:

```
- cd ../blt-project
- phpunit tests/phpunit --group blt-project
- cd ../blt 
- phpunit tests/phpunit --exclude-group blt-project
``` 

The `blt-project` tests can then rely on a $this->project directory variable that refers to `../../../../../../../` given that the actual `TestBase` will live in `vendor/acquia/blt/tests/phpunit/src`.